py/selenium/webdriver/__init__.py

```diff
@@ -25,7 +25,7 @@
 from .edge.webdriver import WebDriver as Edge  # noqa
 from .edge.webdriver import WebDriver as ChromiumEdge  # noqa
 from .edge.options import Options as EdgeOptions  # noqa
-from .opera.webdriver import WebDriver as Opera  # noqa
+from .opera.webdriver import WebDriver as Opera  # noqa  # Todo: gh-10379 (Remove for 4.3)
```

Comment:
Are these `Todo:` necessary for user visibility or simply a note for devs? If the latter than let's not have them in there, we'll know what to delete. :)

Reply:
seems fair; will remove